src/content/docs/fr/guides/deploy/zerops.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Déployez votre site Astro depuis Zerops
+description: Comment déployer votre site Astro sur le web en utilisant Zerops.
+type: deploy
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Steps } from '@astrojs/starlight/components';
+
+[Zerops](https://zerops.io/) est une plateforme cloud "dev-first" qui peut être utilisée pour déployer des sites Astro statiques et SSR.
```

Comment:
Even though we tend to use English terms, I think it's better to translate for those less comfortable with English.
```suggestion
[Zerops](https://zerops.io/) est une plateforme cloud axée sur le développement qui peut être utilisée pour déployer des sites Astro statiques et SSR.
```

Reply:
ah merci, je ne savais pas si je devais le traduire ou pas vu que c'était entre guillemets

Reply:
I don't see quotes [in the source](https://github.com/withastro/docs/blob/98b14771d3886a8dbb01aec65d4c3467b5813b58/src/content/docs/en/guides/deploy/zerops.mdx?plain=1#L10) but even with quotes I tend to translate (and removes the quotes if it makes sense in French). That's my way of seeing things... maybe it's not the best way.  😉